### PR TITLE
import msmtools

### PR DESCRIPTION
**Pull request by marscher**

### CODE REVIEW

**Comment by synapticarbors** [comment]
I'm not sure if a separate license file should be in the recipe directory. Since `LICENSE.txt` is in `MANIFEST.in` it should be copied over into the pkg correctly from the source during the build.

File: recipes/msmtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython >=0.24
+    - numpy x.x
+    - scipy
+    - six
+    - decorator
```
Comment (ericdill) [inline recipes/msmtools/meta.yaml]: While scipy/six/decorator are listed in the `install_requires` bit of the setup.py, the `--single-version-externally-managed` flag causes setup.py to ignore those packages.  As such, you can remove them from the build deps of the meta.yaml.

File: recipes/msmtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - msmtools 
+
+about:
+  home: http://github.com/markovmodel/msmtools
+  license: LGPL-3 
+  license_file: LICENSE.txt
+  summary: 'msmtools is an open source collection of algorithms for the estimation and analysis of discrete state space Markov chains via Markov state models (MSM)'
```
Comment (marscher) [inline recipes/msmtools/meta.yaml]: agreed. Changed that.

File: recipes/msmtools/meta.yaml (hunk 1 above)
Comment (marscher) [inline recipes/msmtools/meta.yaml]: I need only numpy and python, I see.

**Comment by ericdill** [comment]
It is not clear to me why travis timed out.  Given that appveyor and circle-ci succeeded, I am confident that this would have been fine on os-x too.  @jakirkham or @ocefpaf is there any reason why travis-ci would be acting funny?  Maybe one of you could restart the travis job? https://travis-ci.org/conda-forge/staged-recipes/builds/161913207

File: recipes/msmtools/meta.yaml [hunk 3]
```diff
@@ -38,8 +35,10 @@ test:
 about:
   home: http://github.com/markovmodel/msmtools
   license: LGPL-3 
-  license_file: LICENSE.txt
```
Comment (ericdill) [inline recipes/msmtools/meta.yaml]: You still want to include this line though.  That's how conda-build knows what the license file is.  Please re-include this

**Comment by marscher** [comment]
Alright, done.

**Comment by marscher** [comment]
Thank you for the review process! I hope learned enough concepts of conda-forge to add the next recipes without bothering you so much :)

**Comment by ericdill** [comment]
Yay travis is happy again. Alright I think this is done and good to go.  Final review @jakirkham @ocefpaf ?

**Comment by ericdill** [comment]
> Thank you for the review process! I hope learned enough concepts of conda-forge to add the next recipes without bothering you so much :)

Of course!  Worth noting is that my recipes still require review from someone else...  @jakirkham usually finds at least one thing I need to change :grinning:

**Comment by ocefpaf** [comment]
Thanks @marscher for the recipe and @ericdill for the awesome review!

**Pull request merged**